test/testcases/resource.exp

```diff
@@ -712,11 +712,7 @@ INFO: Trace set, restart p0 to trace the stop operation
           <nvpair id="cg-meta_attributes-target-role" name="target-role" value="Started"/>
         </meta_attributes>
         <group id="g">
-          <primitive id="p0" class="ocf" provider="pacemaker" type="Dummy">
-            <meta_attributes id="p0-meta_attributes">
-              <nvpair id="p0-meta_attributes-target-role" name="target-role" value="Started"/>
-            </meta_attributes>
-          </primitive>
+          <primitive id="p0" class="ocf" provider="pacemaker" type="Dummy"/>
```

Comment:
There is a group g containing the primitive p0. The primitive p0 has a attribute Started=true. When setting the Started=true for the group g it will delete the Started=true from the p0.